libs/comparison_rules.py

```
@@ -0,0 +1,123 @@
+
+        Returns:
+        None. The result is stored in the context object.
+        """
+        source = pd.DataFrame(c.m.source)
+        target = pd.DataFrame(c.m.target)
+        results = (~target.isin(source).all(axis=1))
+        if results.all():
+            print(f"Target Row(s) Not Found In Source Data")
+            c.s.result = True
+        else:
+            print(f"Target Row(s) Not Found In Source Data")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)